plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryOutputTableHandle.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.connector.ConnectorOutputTableHandle;
+import io.trino.spi.connector.SchemaTableName;
+import io.trino.spi.type.Type;
+
+import java.util.List;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.util.Objects.requireNonNull;
+
+public class BigQueryOutputTableHandle
+        implements ConnectorOutputTableHandle
+{
+    private final SchemaTableName schemaTableName;
+    private final List<String> columnNames;
+    private final List<Type> columnTypes;
+
+    @JsonCreator
+    public BigQueryOutputTableHandle(
+            @JsonProperty("schemaTableName") SchemaTableName schemaTableName,
```

Comment:
Should this be `SchemaTableName` or the remote mapped names?

Reply:
It should be "remote", but a logic to lookup a remote schema name was missing. Could you take a look at https://github.com/trinodb/trino/compare/28038ec2b874963631ae476ca1b9cdcef36e0462..cac3a485decbaebdada4a356c0b50bab68ca34b7?